app/gui/src/authentication/service.ts

```diff
@@ -118,19 +117,18 @@ export interface AuthService {
 export function useInitAuthService(authConfig: AuthConfig): AuthService {
   const { supportsDeepLinks } = authConfig
 
-  const logger = useLogger()
-  const { router } = useRouter()
+  // TODO[ao] Do we want logger in Vue?
+  // const logger = useLogger()
```

Comment:
When will this be addressed--should it have an issue?

Reply:
I decided that console is preferred for now. The only gains of logger I saw before is that we could forbid `console` so all debug logs were caught before merge. But I just restricted `console.trace` and `console.debug`, so I will use these for debugging now.